WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListActivity.kt

```diff
@@ -157,7 +165,27 @@ class PostsListActivity : AppCompatActivity(),
 
     private fun setupContent() {
         authorSelection = findViewById(R.id.post_list_author_selection)
+
+        val elevationOverlayProvider = ElevationOverlayProvider(this)
+        val appbarElevation = resources.getDimension(R.dimen.appbar_elevation)
+        val appBarColor = elevationOverlayProvider.compositeOverlayIfNeeded(
+                this.getColorFromAttribute(R.attr.wpColorAppBar),
+                appbarElevation
+        )
+
+        appbar.setBackgroundColor(appBarColor)
```

Comment:
I'm not sure I understand what this does. It looks like even without this line, the app bar still looks the same. 🤔 

Without | With this line
--------|-------
<img width="396" alt="without background color" src="https://user-images.githubusercontent.com/198826/71103545-8ede9c80-2177-11ea-91a0-28ac9d284bfe.png"> |       <img width="396" alt="with background color set" src="https://user-images.githubusercontent.com/198826/71103544-8ede9c80-2177-11ea-8195-a08b3a1c9953.png">

Am I looking at the wrong thing? 😅

Reply:
Yeah, you are right! It was supposed to fix a shadow issue in light mode, but I guess some subsequent changes in layout fixed it instead :)